src/components/GameItem.js

```diff
@@ -3,23 +3,43 @@ import '../App.css';
 import ItemIcons from '../ItemIcons.js';
 import PropTypes from 'prop-types';
 
+
 class GameItem extends Component {
-  propTypes = {
+  constructor(props){
+    super(props)
+    this.state = {
+      spotted: false
+    }
+  }
+
+  static propTypes = {
     height: PropTypes.number.isRequired,
     layer: PropTypes.number.isRequired,
   }
 
+  onItemClick = () => {
```

Comment:
It's confusing to have these named so similarly. Could you give it a different name that is easier to differentiate?